Chart.mqh

```diff
@@ -693,10 +693,10 @@ class Chart : public Market {
      * Calculates pivot points in different systems.
      */
     static void CalcPivotPoints(string _symbol, ENUM_TIMEFRAMES _tf, ENUM_PP_METHOD _method, double &PP, double &S1, double &S2, double &S3, double &S4, double &R1, double &R2, double &R3, double &R4) {
-      double _open   = iOpen(_symbol, _tf, 1);
-      double _high   = iHigh(_symbol, _tf, 1);
-      double _low    = iLow(_symbol, _tf, 1);
-      double _close  = iClose(_symbol, _tf, 1);
+      double _open   = ::iOpen(_symbol, _tf, 1);
```

Comment:
You should use `Chart::iOpen` instead, as `::iOpen` won't exist in MQL5?

Reply:
Fixed